phpBB/bin/phpbbcli.php

```diff
@@ -68,6 +68,11 @@
 $phpbb_container->get('request')->enable_super_globals();
 require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
 
+if (is_file($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php'))
+{
+	require_once($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php');
```

Comment:
About sanitization, I think it has to be done on insertion, not on usage. And because we don't provide any interface to do that (only the config:set command which is a tool for advanced user). I think we can assume that the board owner can take its responsibilities.

About making it configurable... I think it's necessary because some installation wouldn't give write access to anyone in some random places but concentrate them in a specific subfolder. We have the same issue for ext/ composer-ext.json and composer-ext.lock. But for them we can't do anything for now. IIRC the composer-ext.json has to be in the root directory otherwise the custom installers (the one installed phpBB ext/styles/lang pack in ext/ styles/ and languages/) will fail. And for ext/ it's related to this composer issue and the core isn't ready yet to put the extensions in a random place (but it could a matter of adding a new config setting)